CONTRIBUTING.md

```diff
@@ -47,24 +51,15 @@ As of Go 1.9 this is simplified to `go test ./...`.
 
 ## Manual Testing against Exercism
 
-You can build whatever is in your local, working copy of the CLI without overwriting your existing Exercism
-CLI installation by using the `go build` command:
+If you want to test your changes while doing your everyday exorcism work you could do:
 
-```
-go build -o testercism exercism/main.go
-```
-
-This assumes that you are standing at the root of the exercism/cli repository checked out locally, and it will put a binary named `testercism` in your current working directory.
-
-You can call it whatever you like, but `exercism` would conflict with the directory that is already there.
-
-Then you call it with `./testercism`.
+On Unices:
 
-You can always put this in your path if you want to run it from elsewhere on your system.
+- `cd $GOPATH/src/github.com/exercism/cli/exercism && go build -o exercism main.go && mv exercism ~/bin`
```

Comment:
I think it's important to call it something other than `exercism` when running a test build. I use `testercism` but it could be anything.

This will:

- put test configuration in a different directory (on *nix: `~/.config/testercism`)
- configure the workspace to a different default than what people use when doing Exercism exercises (e.g. `$HOME/testercism`)

That way the test config and test exercises won't clobber the existing configuration and exercises.

This is typically good even if you want to talk to the live server, since often you'll mess around with setting a different token just to see what error messages look like, etc.

Reply:
👍